docs/reference/setup/install/get-fingerprint.asciidoc

```diff
@@ -0,0 +1,44 @@
+==== Connect clients to {es}
```

Comment:
Apologies in advance, but I'll suggest another refactoring. I find the current structure quite confusing for users as is

```
==== Connect clients to {es}

When you start {es} for the first time, TLS is configured for the HTTP layer and a CA certificate
is generated and stored on disk. The hex-encoded SHA-256 fingerprint of that certificate is also printed
in the terminal. Any clients that connect to {es} such as the 
https://www.elastic.co/guide/en/elasticsearch/client/index.html[{es} Clients],
{beats}, {ls}, and {fleet} need to validate that they trust the certificate that {es} uses for HTTPS and they 
can do so by using either the fingerprint of the CA certificate or the CA certificate itself.

[discrete]
===== Use the CA fingerprint

Copy the fingerprint value that {es} printed to the terminal on startup and configure your client to use this
fingerprint to establish trust when it connects to {es}.

If the auto-configuration process already completed, you can still obtain the 
fingerprint of the security certificate by running the following command. The path is to the
auto-generated CA certificate for the HTTP layer.

[source,sh]
----
openssl x509 -fingerprint -sha256 -in config/tls_auto_config_<timestamp>/http_ca.crt
----

`<timestamp>`:: The timestamp of when the auto-configuration process created the security files directory.

The command returns the the security certificate, including the fingerprint.
The `issuer` should be `Elasticsearch security auto-configuration HTTP CA`.

[source,sh]
----
issuer= /CN=Elasticsearch security auto-configuration HTTP CA
SHA256 Fingerprint=<fingerprint>
----

[discrete]
===== Use the CA certificate

If your library or client doesn't support a method of validating the fingerprint, the 
auto-generated CA certificate is created in the
`config/tls_auto_config_<timestamp>` directory on each {es} node. Copy the
`http_ca.crt` file to your machine and configure your client to use this
certificate to establish trust when it connects to {es}.
```

Reply:
Thanks @jkakavas 👍 I updated the sections to reflect your suggested changes. Let me know if you're +1 on merging given the recent updates.